### PR TITLE
Update README.md for stackdriverexporter.

### PR DESCRIPTION
**Pull request by dengliming**
**Description:** Update README.md for stackdriverexporter.
for [https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/318](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/318)

### CODE REVIEW

**Comment by ocervell** [comment]
As long as we're documenting this: I had a hard time with the `resource_mappings`, especially the `source_type` and `target_type` here. Can we switch to a real world example with actual Stackdriver resources like `k8s_container`, `global`, or `gce_instance` to make it clearer ? Thanks 👍

File: exporter/stackdriverexporter/README.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
 # Stackdriver Exporter
 
-To be added.
\ No newline at end of file
+This exporter can be used to send metrics to Stackdriver.
```
Comment (nilebox) [inline exporter/stackdriverexporter/README.md:3]: Note: Stackdriver has been rebranded to Cloud Operations Suite at Google.
```suggestion
This exporter can be used to send metrics to Google Cloud Monitoring and Trace (formerly known as Stackdriver).
```

File: exporter/stackdriverexporter/README.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+
+The following configuration options are supported:
+
+- `project` (required): ProjectID is the identifier of the Stackdriver.
```
Comment (nilebox) [inline exporter/stackdriverexporter/README.md:7]: I was able to use `stackdriver` exporter with no config parameters at all, so none of the parameters in the list are required.

Specifically for `project` it's worth mentioning that the project will be automatically detected in the GCP environment unless explicitly overridden via this parameter.

**Review by nilebox — COMMENTED** [review]
Thanks for making this PR, it's super useful!
Please address a few minor comments.

File: exporter/stackdriverexporter/README.md (hunk 2 above)
Comment (dengliming) [reply exporter/stackdriverexporter/README.md]: @nilebox Thanks for helping. fixed at lastest commit.

File: exporter/stackdriverexporter/README.md [hunk 3]
```diff
@@ -1,3 +1,47 @@
 # Stackdriver Exporter
 
-To be added.
\ No newline at end of file
+This exporter can be used to send metrics and traces to Google Cloud Monitoring and Trace (formerly known as Stackdriver) respectively.
+
+The following configuration options are supported:
+
+- `project` (optional): ProjectID is the identifier of the Stackdriver.
```
Comment (nilebox) [inline exporter/stackdriverexporter/README.md:7]:
```suggestion
- `project` (optional): ProjectID is the identifier of the GCP project.
```

File: exporter/stackdriverexporter/README.md (hunk 2 above)
Comment (nilebox) [reply exporter/stackdriverexporter/README.md]: Thanks! Also need to rephrase the description, see new comment.

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**